### PR TITLE
added new team member

### PR DESCRIPTION
**Pull request by chamodperera**
preview link-https://chamodperera.github.io/fossasia.org/team/

team member

### CODE REVIEW

**Comment by chamodperera** [comment]
@mariobehling please accept

**Comment by kumuditha-udayanga** [comment]
LGTM 😇

**Comment by mariobehling** [comment]
GitHub link is wrong. We also need a better photo, please. @atm1504 Please help with a better quality image.

**Comment by atm1504** [comment]
> GitHub link is wrong. We also need a better photo, please. @atm1504 Please help with a better quality image.

Yah sure. I am giving the required data

File: team/index.html [hunk 1]
```diff
@@ -1472,6 +1472,22 @@ <h1>Developers</h1>
 						<span>Android Developer</span>
 					</div>
 				</div>
+				<div class="col-md-3 col-sm-6 speaker-column">
+					<div class="speaker">
+						<div class="image-holder">
+							<img class="background-image" alt="Amrtya Mondal" src="../img/team/amrtyamondal.png">
+							<div class="hover-state text-center preserve3d">
+								<div class="social-links vertical-align">
+									<a href="<a href="https://github.com/neel1998" target="blank"><i class="icon fa fa-github"></i></a>" target="blank"><i class="icon fa fa-github"></i></a>
```
Comment (atm1504) [inline team/index.html]: Wrong GitHub link. Please change it. The correct link should be ```https://github.com/atm1504```.
Wrong spelling in alt tag

File: team/index.html [hunk 2]
```diff
@@ -1472,6 +1472,22 @@ <h1>Developers</h1>
...
+									<a href="https://www.facebook.com/atm1504" target="blank"><i class="icon social_facebook"></i></a>
+									<a href="https://twitter.com/AmartyaMondal7" target="blank"><i class="icon social_twitter"></i></a>
+								</div>
+							</div>
+						</div>
+						<span class="speaker-name">Amrtya Mondal</span>
```
Comment (atm1504) [inline team/index.html]: Spelling wrong. Correct spelling is ```Amartya Mondal```

**Review by atm1504 — CHANGES_REQUESTED** [review]

**Comment by atm1504** [comment]
atm
@chamodperera please use this image

**Comment by chamodperera** [comment]
@atm1504  @mariobehling  Changes have been done

git3

**Review by anishagg17 — CHANGES_REQUESTED** [review]
I think you should squash  all your commits.

**Review by liveHarshit — CHANGES_REQUESTED** [review]
You're updating one team member but adding three photos, why? Also update the PR title to be specific, whom you are adding.

**Pull request closed without merging**